tests/functionaltests/UXTestHelpers.mm

```diff
@@ -106,4 +114,27 @@
     }
 }
 
+//
+// UXEvent methods
+//
+void UXEvent::Signal() {
+    [_condition lock];
+
+    _signaled = true;
+    [_condition signal];
+
+    [_condition unlock];
+}
+
+bool UXEvent::Wait(int timeOutInSeconds) {
```

Comment:
agree wait is more aligned, it is just return is unclear 

if read 

ASSERT_TRUE_MSG(uxEvent->Wait(c_testTimeoutInSec), "FAILED: Waiting for property changed event timed out!");

I feel uxEvent->Wait isn't clear enough for reader to know what the expectation is. should we expect true or false. 

---
In reply to: [98268635](https://github.com/Microsoft/WinObjC/pull/1855#discussion_r98268635) [](ancestors = 98268635,98267209)